### ISSUE TITLE
Restores closed editors

### ISSUE THREAD

**Issue by ldelossa** [aggrhm/vscode-workviews#2]
hello, 

it seems like any closed editor is restored once moving away from, and then back to, a saved workview.

**Comment by aggrhm** [comment]
Yes I'm aware that this can sometimes happen due to the nature of how I have to access open editors within VS Code.

For some reason, VS Code does not give access to all the open editors, only the currently visible ones. To get around this, I listen to a "document closed" event that does not fire reliably. Eventually, hopefully, VS Code will give access to all open editors so that I can just save them when a Workview is closed.

In the meantime, a quick fix that I've added is to use the "Restore pinned only" setting to only restore pinned documents or documents that were last visible. It works pretty well.
